data/coalition.txt

```diff
@@ -735,3 +735,98 @@ phrase "friendly heliarch"
 		"we have committed."
 		"we do serve."
 		"we do vow."
+
+phrase "hostile heliarch"
+	word
+		"Destroying "
+		"Betraying "
+		"Dishonoring "
+		"Ruining "
+		"Invading "
+		"Attacking "
+		"Conquering "
+		"Oppressing "
+		"Backstabbing "
+	word
+		"the Coalition "
```

Comment:
```
word
    " "
word
    "the"
    "our"
word
    " "
```

Reply:
No, `...out citizens of the coalition...` `...the just government of merit...` don't make complete sense. An argument can be made for both but they are a little odd. Having "the" instead of "our" on the last four would sound like they think that any other government is unjust (or something).

Reply:
That's just an argument to keep going and diversify the hail though - it's not really an argument against this. You can make a set of phrases that work with with `the` and a set that work with `our`, and a set that works with both, etc.

Reply:
That would require 3 entire sets of hails that are almost the same apart from a couple of lines in the middle.

Reply:
I mean to use a sub-`phase`, e.g. `phrase "heliarch possessive unions"`, which is all phrases that pair well with `our` as a lead-in, and then you call that within this actual hostile hail. At minimum we both can agree that the last bit's duplication is easily solved by moving "will" and "shall" to a `word` token:

```
  "will live on"
  "shall live on"
  "will not be destroyed"
  "shall not be destroyed"
```
should absolutely be
```
word
  "will"
  "shall"
word
  " "
word
  "live on"
  "not be destroyed"

Reply:
Should be okay now